test/extended/default_forcePull_test.go

```diff
@@ -0,0 +1,262 @@
+// +build default
+
+package extended
+
+import (
+	"fmt"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"time"
+)
+
+var (
+	fpTrueS2I         = filepath.Join("fixtures", "forcepull-true-s2i.json")
```

Comment:
I did a combo of things here.  With some refactoring, I've reduced the needed scope for the fp* variables for the various json file paths.  I've moved those 6 variables into the precise scopes in which they are needed vs. having them global.

I made s2iDockBldr and custBldr constants and moved the hex IDs into a map.

Also removed direct refs to these in image_helper.go and docker.go.